api/v2/api.go

```diff
@@ -216,18 +214,20 @@ func (api *API) getAlertsHandler(params alert_ops.GetAlertsParams) middleware.Re
 		// are no alerts present
 		res = open_api_models.GettableAlerts{}
 		ctx = params.HTTPRequest.Context()
+
+		logger = log.With(api.logger, "handler", "alerts", "methods", "GET")
```

Comment:
do we have to hardcode those info?

Reply:
You could probably extract some of them from `params.HTTPRequest`.

Reply:
Thanks for the feedback! I've updated the code.